### PR TITLE
fix(function-tree): make websocket work cross browser and node

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: packages/function-tree/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "chalk": "^1.1.3",
     "eventemitter3": "^2.0.2",
+    "universal-websocket-client": "^1.0.1",
     "ws": "^1.1.1"
```
Comment (henri-hulski) [inline packages/function-tree/package.json]: ws is already dependency of universal-websocket-client (version 2.2.0)

**Comment by Guria** [comment]
Hmm. Why not to pass ws instance to provider config?

**Comment by christianalfoni** [comment]
@Guria Too much configuration in my opinion. This should just work... and it does now :)

File: packages/function-tree/package.json (hunk 1 above)
Comment (christianalfoni) [inline packages/function-tree/package.json]: Good catch, fixed!

**Pull request merged**